### ISSUE TITLE
Accessibility: https://jupyter.org/governance/conduct/reporting_online.html

### ISSUE THREAD

**Issue by luquiceno** [jupyterhub/outreachy#96]
### What page is this for?

https://jupyter.org/governance/conduct/reporting_online.html

### WAVE accessibility report

https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1274454640

![accessibility_ReportingOnlinePage](https://user-images.githubusercontent.com/103491565/196706247-6264248f-5941-4332-809e-98eeeefd56bb.png)

**1**. Language of the  web page
**2**. Button "fullscreen mode" has no text value
**3**. Error with labels
**4**. ARIA label
**5**. Appropriate alternative text an image

### The HTML element to be changed

```html
1. The page doesn’t contain the "lang" attribute in the html element.

<html style="margin-left: 380px;">

2. The button doesn’t have a value text.

<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
    <span class="headerbtn__icon-container">
        <i class="fas fa-expand"></i>
    </span>
</button>

3. Some labels without any text content and 2 multiple form labels.

<input checked="" class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox">
<label for="toctree-checkbox-1">
    <i class="fas fa-chevron-down"></i>
</label>

<input class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox">
<label for="toctree-checkbox-2">
    <i class="fas fa-chevron-down"></i>
</label>

<input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox">
<label for="toctree-checkbox-3">
    <i class="fas fa-chevron-down"></i>
</label>

<div class="header-article__left">
    <label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
        <span class="headerbtn__icon-container">
            <i class="fas fa-bars"></i>
        </span>
    </label>
</div>

<label for="__page-toc" class="headerbtn headerbtn-page-toc">
    <span class="headerbtn__icon-container">
        <i class="fas fa-list"></i>
    </span>
</label>

Multiple labels could be a problem for the assistive technology. A possible solution to this accessibility issue, is to use the attribute "aria-labelledby". 

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" 
    aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" 
    aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">

4. ARIA label

<p aria-level="2" class="caption" role="heading">
    <span class="caption-text">
        Code of Conduct
    </span>
</p>

<p aria-level="2" class="caption" role="heading">
    <span class="caption-text">
        Governance
    </span>
</p>

<p aria-level="2" class="caption" role="heading">
    <span class="caption-text">
        Organizational Structure
    </span>
</p>

<p aria-level="2" class="caption" role="heading">
    <span class="caption-text">
        Organizational Policy
    </span>
</p>

<p aria-level="2" class="caption" role="heading">
    <span class="caption-text">
        Future Governance [work in progress]
    </span>
</p>

5. The text for the attribute "alt" is very short.

<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" alt="CC0" style="border-style: none;">
```

### Your proposed HTML (or CSS), after change

```html
1. 
<html style="margin-left: 380px;" lang="en">

2. 
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" aria-label="fullscreenbutton" data-placement="bottom" title="" data-original-title="Fullscreen mode">
        <span class="headerbtn__icon-container">
        <i class="fas fa-expand"></i>
        </span>
    </button>

3. 
<input checked="" class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox">
<label for="toctree-checkbox-1" title="checkbox-1">
    <i class="fas fa-chevron-down"></i>
</label>

<input class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox">
<label for="toctree-checkbox-2" title="checkbox-2>
    <i class="fas fa-chevron-down"></i>
</label>

<input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox">
<label for="toctree-checkbox-3" title="checkbox-3>
    <i class="fas fa-chevron-down"></i>
</label>

<div class="header-article__left">
    <label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="toggleNavigation" data-original-title="Toggle navigation">
        <span class="headerbtn__icon-container">
            <i class="fas fa-bars"></i>
        </span>
    </label>
</div>

<label for="__page-toc" class="headerbtn headerbtn-page-toc" title="pageTableContents">
    <span class="headerbtn__icon-container">
        <i class="fas fa-list"></i>
    </span>
</label>

Multiple form labels

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" 
    aria-labelledby="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" 
    aria-labelledby="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">

4. 
<p role="heading" aria-level="2" class="caption">
    <span class="caption-text">
        Code of Conduct
    </span>
</p>

<p role="heading" aria-level="2" class="caption">
    <span class="caption-text">
        Governance
    </span>
</p>

<p role="heading" aria-level="2" class="caption">
    <span class="caption-text">
        Organizational Structure
    </span>
</p>

<p role="heading" aria-level="2" class="caption">
    <span class="caption-text">
        Organizational Policy
    </span>
</p>

<p role="heading" aria-level="2" class="caption">
    <span class="caption-text">
        Future Governance [work in progress]
    </span>
</p>

Using headings <h2>, rather than using "role" and "aria-level".

<h2 class="caption">
    <span class="caption-text">
        Code of Conduct
    </span>
</h2>

<h2 class="caption">
    <span class="caption-text">
        Governance
    </span>
</h2>

<h2 class="caption">
    <span class="caption-text">
        Organizational Structure
    </span>
</h2>

<h2 class="caption">
    <span class="caption-text">
        Organizational Policy
    </span>
</h2>

<h2 class="caption">
    <span class="caption-text">
        Future Governance [work in progress]
    </span>
</h2>

5.
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" alt="Creative Commons Zero license CC0" style="border-style: none;">
```

**Comment by minrk** [comment]
Thanks for the suggestions!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
